lib/tokens/css/variables.css

```diff
@@ -99,23 +99,21 @@
   --eds-color-highlight-400: #9DFFA4;
   --eds-color-highlight-500: #00F1FF;
   --eds-color-highlight-600: #CFC9FF;
-  --eds-size-font-h1: 3rem; /* Font size for Heading 1 sized text */
-  --eds-size-font-h2: 2.5rem; /* Font size for Heading 2 sized text */
-  --eds-size-font-h3: 2rem; /* Font size for Heading 3 sized text */
-  --eds-size-font-h4: 1.5rem; /* Font size for Heading 4 sized text */
-  --eds-size-font-h5: 1.125rem; /* Font size for Heading 5 sized text */
-  --eds-size-font-base: 16px; /* Base font size for design system in px */
-  --eds-size-font-body: 1rem; /* Default font size for body text */
-  --eds-size-font-sm: 0.875rem; /* Font size for small body text */
-  --eds-size-font-xs: 0.75rem; /* Font size for xsmall body text */
-  --eds-size-line-height-h1: 4rem; /* Line height for Heading 1 sized text */
-  --eds-size-line-height-h2: 3.5rem; /* Line height for Heading 2 sized text */
-  --eds-size-line-height-h3: 2.5rem; /* Line height for Heading 3 sized text */
-  --eds-size-line-height-h4: 2rem; /* Line height for Heading 4 sized text */
-  --eds-size-line-height-h5: 1.5rem; /* Font size for Heading 5 sized text */
-  --eds-size-line-height-body: 1.5rem; /* Default line height for body text */
-  --eds-size-line-height-sm: 1.25rem; /* Line height for small body text */
-  --eds-size-line-height-xs: 1rem; /* Line height for xsmall body text */
+  --eds-font-family-primary: 'Arimo', sans-serif;
+  --eds-font-family-secondary: 'Open Sans', sans-serif;
+  --eds-font-size-11: 0.688rem;
+  --eds-font-size-12: 0.75rem;
+  --eds-font-size-14: 0.875rem;
+  --eds-font-size-16: 1rem;
+  --eds-font-size-18: 1.125rem;
+  --eds-font-size-24: 1.5rem;
+  --eds-font-size-32: 2rem;
+  --eds-font-size-40: 2.5rem;
+  --eds-font-size-base: 16px; /* Base font size for design system in px */
+  --eds-font-weight-light: 400;
+  --eds-font-weight-medium: 400; /* Arimo weight, should be updated to 500 */
```

Comment:
Nit: could you add "when we switch to Graphik" at the end of this comment and the one on the next line?